x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/WaterfallWithSummmary/TraceSummary.tsx

```diff
@@ -0,0 +1,255 @@
+import { i18n } from '@kbn/i18n';
+import { idx } from '@kbn/elastic-idx';
+import moment from 'moment';
+import euiLightVars from '@elastic/eui/dist/eui_theme_light.json';
+import styled from 'styled-components';
+import { Transaction } from '../../../../../typings/es_schemas/ui/Transaction';
+import { asTime, asPercent } from '../../../../utils/formatters';
+import { NOT_AVAILABLE_LABEL } from '../../../../../common/i18n';
+import { isRumAgentName } from '../../../../../common/agent_name';
+import { ErrorCountBadge } from './ErrorCountBadge';
+import { truncate } from '../../../../style/variables';
+
+// TODO: Light/Dark theme (@see https://github.com/elastic/kibana/issues/44840)
+const theme = euiLightVars;
+
+const Heading: React.FunctionComponent = () => (
+  <EuiTitle size="xs">
+    <h5>
+      {i18n.translate('xpack.apm.transactionDetails.traceSampleTitle', {
+        defaultMessage: 'Trace sample'
+      })}
+    </h5>
+  </EuiTitle>
+);
+
+const Item = styled('span')`
+  white-space: nowrap;
+`;
+
+interface TimestampProps {
+  transactionTimestamp: string;
+}
+const Timestamp: React.FunctionComponent<TimestampProps> = ({
+  transactionTimestamp
+}) => {
+  const timestamp = moment(transactionTimestamp).format(
+    'MMMM Do YYYY, HH:mm:ss.SSS'
+  );
+
+  return (
+    <Item>
+      <EuiToolTip
+        content={i18n.translate('xpack.apm.transactionDetails.timestampLabel', {
+          defaultMessage: 'Timestamp'
+        })}
+      >
+        <>{timestamp}</>
+      </EuiToolTip>
+    </Item>
+  );
+};
+
+const SeparatorText = styled(EuiText)`
+  color: ${theme.euiColorMediumShade};
+  display: inline;
+`;
+const Separator: React.FunctionComponent = () => (
+  <>
+    {' '}
+    <SeparatorText aria-hidden={true}>|</SeparatorText>{' '}
+  </>
+);
+
+interface DurationProps {
+  duration: number;
+  totalDuration?: number;
+}
+const Duration: React.FunctionComponent<DurationProps> = ({
+  duration,
+  totalDuration
+}) => {
+  totalDuration = totalDuration || duration;
+  const isOver100 = duration > totalDuration;
+  const percentOfTrace = isOver100
+    ? '>100%'
+    : asPercent(duration, totalDuration, '');
+  const percentOfTraceText = i18n.translate(
+    'xpack.apm.transactionDetails.percentOfTrace',
+    {
+      defaultMessage: '{value} of trace',
+      values: { value: percentOfTrace }
+    }
+  );
+
+  return (
+    <Item>
+      <EuiToolTip
+        content={i18n.translate('xpack.apm.transactionDetails.durationLabel', {
+          defaultMessage: 'Duration'
+        })}
+      >
+        <>{asTime(duration)}</>
+      </EuiToolTip>{' '}
+      {isOver100 ? (
+        <EuiToolTip
+          content={i18n.translate(
+            'xpack.apm.transactionDetails.percentOfTraceLabelExplanation',
+            {
+              defaultMessage:
+                'The % of trace exceeds 100% because this transaction takes longer than the root transaction.'
+            }
+          )}
+        >
+          <>({percentOfTraceText})</>
```

Comment:
Is the fragment needed?
```suggestion
          ({percentOfTraceText})
```

Reply:
Oh - is it because it's required by `EuiToolTip`? Then it makes sense (although EUI should just accept strings in the first place)

Reply:
Opened https://github.com/elastic/eui/issues/2292 to track this.